.github/workflows/ci.yml

```diff
@@ -258,7 +258,7 @@ jobs:
           toolchain: stable
           override: true
           components: clippy
-      - uses: Swatinem/rust-cache@v1
+      # - uses: Swatinem/rust-cache@v1
```

Comment:
If you want to comment out, please add some information on why do this.

Reply:
Added description.

https://github.com/openrr/openrr/blob/1d1885cf304d110bee27401922c4374aceeaae68/.github/workflows/ci.yml#L261-L265